test/Directory.Build.props

```diff
@@ -2,8 +2,8 @@
   <Import Project="..\Directory.Build.props" />
 
   <PropertyGroup>
-    <TargetFrameworks Condition="'$(OS)' == 'Windows_NT'">net452;net461;netcoreapp2.1;netcoreapp3.0;netcoreapp3.1</TargetFrameworks>
-    <TargetFrameworks Condition="'$(OS)' != 'Windows_NT'">netcoreapp2.1;netcoreapp3.0;netcoreapp3.1</TargetFrameworks>
+    <TargetFrameworks Condition="'$(OS)' == 'Windows_NT'">net452;net461;netcoreapp2.1;netcoreapp3.0;netcoreapp3.1;net5.0</TargetFrameworks>
```

Comment:
Since net5.0 is not GA yet, should we include `net5.0` based on a build property?

Reply:
That's a very good point. I did the whole thing with VS Preview, and forgot it wouldn't work with the "normal" version

Reply:
Added conditions. Now net5.0 should only be enabled when using VS Preview.